api/management/commands/sync_molnix.py

```diff
@@ -242,6 +266,14 @@ def sync_deployments(molnix_deployments, molnix_api, countries):
         personnel.country_to = country_to
         country_from = None
         personnel.surge_alert = surge_alert
+        personnel.appraisal_score = appraisal_score
+        if gender == 'male':
```

Comment:
That part could be nicer.

Reply:
@szabozoltan69 let's just keep this a CharField for now instead of enforcing a CHOICES